common/interpreter/wawaka_wasm/WasmCryptoExtensions.cpp

```diff
@@ -645,3 +648,160 @@ extern "C" bool verify_sgx_report_wrapper(
 
     return false;
 }
+
+/* ----------------------------------------------------------------- *
+ * NAME: parse_sgx_report_wrapper(
+ * ----------------------------------------------------------------- */
+extern "C" bool parse_sgx_report_wrapper(
```

Comment:
How will this function be used? I assume this will be useful for cases in which a contract may wish to store the contract enclave's SGX report.

Reply:
the test shows how to invoke and what the results look like. my expectation is that we would use this to verify configid and mrenclave from other sgx enclaves, e.g. this will be used to verify an attestation from a gramine service.